cubic-server/Server.hpp

```diff
@@ -17,15 +17,12 @@
 #include "protocol_id_converter/itemConverter.hpp"
 
 #include "Permissions.hpp"
+#include "options.hpp"
```

Comment:
This include is not used in this file, do you want to add more NODISCARD macro ?